x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -88,15 +84,28 @@ export class Distribution extends Component<Props> {
 
     return (
       <div>
-        <HeaderSmall
-          css={`
-            display: flex;
-            align-items: center;
-          `}
-        >
-          <span>Response time distribution</span>
-          <SamplingTooltip />
-        </HeaderSmall>
+        <EuiTitle size="s">
```

Comment:
I would prefer that this and the other two chart titles all get `xs` rather than `s`, if at all possible.